rmd_files/RNAseq_report.Rmd

```diff
@@ -4473,7 +4473,7 @@ if ( nrow(fusion_annot_top) > 0 ) {
 ```
 
 ```{r genomic_view_circos_table_fusions, comment = NA, message=FALSE, warning=FALSE, eval = runFusionChunk}
-if ( nrow(fusion_annot_top) > 0 ) {
+if ( nrow(fusion_annot_top) > 0 && runArribaChunk ) {
```

Comment:
As before, possible we should set this one to "runPizzlyChunk" rather than "runArribaChunk"